haystack/pipelines/base.py

```diff
@@ -1932,6 +1932,8 @@ def print_eval_report(
             "document_id_or_answer",
         ] = "document_id_or_answer",
         answer_scope: Literal["any", "context", "document_id", "document_id_and_context"] = "any",
+        wrong_examples_fields: Optional[List[str]] = None,
```

Comment:
We need some good default values here. Look at my previous review. I left a suggestion for that.